### PR TITLE
feat(query): indents that reflect the formatter

### PR DESCRIPTION
**Pull request by ribru17**
Indentation queries that reflect the indentation of lucario's query formatter. Only applicable if `vim.opt_local.lispoptions = 'expr:1'`

### CODE REVIEW

**Comment by lucario387** [comment]
I will not be able to check this for a while, so can you write some test files to reflect that? I think it's great as is right now.

**Comment by ribru17** [comment]
Ok I think I have everything working now, but the queries are needlessly complicated and the indentation was basically fine before. I think this nonsense was a waste of time and I need to learn how to not procrastinate and go to bed. Feel free to close if too much extra code, I'm not even sure if I covered all edge cases (and 99% of users will probably be using the default lisp mode indentation anyway?)

**Comment by lucario387** [comment]
We are the 1% :) 
That's more than enough tbh

**Review by lucario387 — APPROVED** [review]
Updated the queries to better reflect the intention.

Great thanks for the work!

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -46,5 +46,5 @@ jobs:
 
       - name: Lint
         run: |
-          nvim -l scripts/format-queries.lua
+          nvim -l scripts/format-queries.lua queries
```
Comment (clason) [inline .github/workflows/lint.yml:49]:
```suggestion
          nvim -l scripts/format-queries.lua
```

You need to rebase; I changed the script to default to current dir.

**Comment by lucario387** [comment]
@clason Can I do a "partial" revert for the formatting scripts to only target `queries`? It will help if there are extra scheme/query test files in the future.

File: .github/workflows/lint.yml (hunk 1 above)
Comment (lucario387) [reply .github/workflows/lint.yml:49]: It also tries to format the test file, so I deliberately do that again

Comment (lucario387) [reply .github/workflows/lint.yml:49]: Maybe there's a better solution here :thinking:

Comment (lucario387) [reply .github/workflows/lint.yml]: Decided to `; format-ignore` pilled that file

**Comment by ribru17** [comment]
Thanks for salvaging this!

**Pull request merged**